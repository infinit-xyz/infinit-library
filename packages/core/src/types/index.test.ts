import { describe, expect, it } from 'vitest'

import * as index from './index'

describe('types/index.ts', () => {
  it('should match the snapshot', () => {
    expect(Object.keys(index)).toMatchSnapshot()
  })
})
