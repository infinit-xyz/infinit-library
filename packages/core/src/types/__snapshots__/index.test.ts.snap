// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`types/index.ts > should match the snapshot 1`] = `
[
  "InfinitCache",
  "OffChainActionCallback",
  "OnChainActionCallback",
  "ContractVerifierCallback",
]
`;
